openapi/control.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Control API</title>
```

Comment:
Given this is an HTML page, and rendered directly, I think this should mention Ably

Reply:
Fixed.